tox.ini

```diff
@@ -1,5 +1,7 @@
 [tox]
-envlist = py27,py34,py35,py36,py37
+envlist =
+    py{34,35,36,37}-pydicom1
+    py27-pydicom{0,1}
```

Comment:
This is awesome!

Reply:
Tox has some really nice support for generative environment definitions-- I never had the chance to play with it before :)